src/main/java/seedu/taskman/logic/commands/Command.java

```diff
@@ -1,15 +1,27 @@
 package seedu.taskman.logic.commands;
 
+import java.util.concurrent.LinkedBlockingDeque;
+
 import seedu.taskman.commons.core.EventsCenter;
 import seedu.taskman.commons.core.Messages;
 import seedu.taskman.commons.events.ui.IncorrectCommandAttemptedEvent;
 import seedu.taskman.model.Model;
 
 /**
  * Represents a command with hidden internal logic and the ability to be executed.
+ * 
+ * Command History:
+ * LinkedBlockingDeque
+ * Front --	---- ---- ---- ---- ---- ---- ---- ---- Back
+ * offerFirst/pollFirst/pop	---	---- ---- ---- ---- offerLast/pollLast
+ * Most recently executed command -- ---- ---- ---- 10th most recently executed command
```

Comment:
I'm not that good at writing comments either, but I think this comment might need some work. 

Maybe more text describing the diagram will help?